wire/hybridbus_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wire_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	. "perun.network/go-perun/wire"
+	"perun.network/go-perun/wire/test"
+)
+
+func TestHybridBus_New(t *testing.T) {
```

Comment:
I am not the biggest fan of such tests. They are quite tedious to maintain and don't bring much value in my opinion.